Samples/DotNetClientConsole/Helpers/RelativityImportEndpoints.cs

```diff
@@ -2,6 +2,7 @@
 // © Relativity All Rights Reserved.
 // </copyright>
 
+#pragma warning disable SA1600 // Elements should be documented

```

Comment:
Why have we added doc comments in other files and ignored that rule here?

Reply:
Documentation of this class added and the rule enabled.